recently-edited-content-widget.php

```diff
@@ -108,6 +108,17 @@ protected static function getQuery($args = array())
             $args['meta_value'] = get_current_user_id();
         }
 
+        if (self::$options['since_date']) {
+            $args['date_query'] = array(
+              array(
+                'after' => self::$options['since_date'],
+                'column' => 'post_modified_gmt'
+              )
+            );
+            // Remove page limit when since_date is set
+            $args['posts_per_page'] = -1;
```

Comment:
Trouble is, with posts_per_page set there's no way for the user to tell if they are seeing all the items in the date range or not (or to view any items beyond the num_pages limit). 

What do you think about adding pagination on the display side? 

I'm happy to remove that line, but for the use case I'm after (reviewing changes before deploying from a staging site to production) it's important that all changes within the date range can be viewed, so I would need to find a different way to make this work.